metadat/no-intro/Nintendo - Super Nintendo Entertainment System.dat

```diff
@@ -4465,7 +4465,7 @@ game (
 	name "Doraemon 4 - Nobita to Tsuki no Oukoku (Japan)"
 	description "Doraemon 4 - Nobita to Tsuki no Oukoku (Japan)"
 	region "Japan"
-	rom ( name "Doraemon 4 - Nobita to Tsuki no Oukoku (Japan).sfc" size 1572864 crc 4097BC01 md5 930CB24C28F2851F8C3A99A588B9FE8B sha1 E9C04238C10240A2C9F8DBD7CF1E3021F01C9C6D )
+	rom ( name "Doraemon 4 - Nobita to Tsuki no Oukoku (Japan).sfc" size 2097152 crc 5B6C63C8 md5 37ED37646C9F206097450A311E1094EA sha1 613D396A49892B9F81FB5A8C395578FA443B3D84 )
```

Comment:
If you're submitting no-into fixes, you may want to submit them upstead to No-Intro themself.

``` suggestion
	rom ( name "Doraemon 4 - Nobita to Tsuki no Oukoku (Japan).sfc" size 1572864 crc 4097BC01 md5 930CB24C28F2851F8C3A99A588B9FE8B sha1 E9C04238C10240A2C9F8DBD7CF1E3021F01C9C6D )
```